css/style.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  min-height: 100vh;
+  margin: 0;
+  background: #191765;
+  background-image: url('/images/portfoltiobg1.png');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  color: #c8c8d4;
+  font-family: Roboto, 'sans serif';
+  line-height: 24px;
+  font-size: 15;
+}
+img {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 40px;
+}
```

Comment:
- [ ] [OPTIONAL] instead of having the background image in the  `body` put in here `wrapper` like this the white big dot doesn't hover on the log and it will be better for the read. 
```css
.wrapper {
  background-image: url('/images/portfoltiobg1.png');
  background-repeat: no-repeat  ;
  display: flex;
  flex-direction: column;
  padding: 20px 40px;
}
```
Note that I use ` background-repeat: no-repeat` this is because the image will be double.